vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -131,36 +149,53 @@
-                currentRoutePath.includes(navbar.url)
-                  ? ''
-                    ? ''
-                    : 'after:tw-w-full after:tw-scale-x-0 after:tw-scale-x-100'
-                  : '',
+                    ' after:tw-absolute after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-w-full after:tw-h-[3px]  after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent',
               ]"
+              @click="
+                navbar.name == 'Contribution'
+                  ? (showContributionMenu = !showContributionMenu)
+                  : ''
+              "
             >
-              <span
-                :class="[
-                  currentRoutePath == navbar.url
-                    ? 'v2-canopas-gradient-text'
-                    : '',
-                ]"
-                class="tw-flex tw-mt-[0.1rem]"
-                >{{ navbar.name }}</span
-              ></router-link
+              {{ navbar.name }}</router-link
             >
+
             <a
               v-else
-              class="tw-inline-block tw-relative tw-mr-[20px] sm:tw-mr-[30px] after:tw-absolute after:tw-w-full after:tw-h-[3px] after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent"
+              class="tw-inline-block tw-relative tw-mr-[20px] sm:tw-mr-[30px] after:tw-absolute after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-w-full after:tw-h-[3px] after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent"
               :href="navbar.url"
               :target="navbar.target"
               >{{ navbar.name }}</a
             >
+            <ul
+              :class="
+                showContributionMenu && navbar.name == 'Contribution'
+                  ? 'tw-mt-8 tw-w-full tw-px-6 tw-bg-white'
+                  : 'tw-hidden tw-overflow-hidden'
+              "
+              class="tw-flex-col tw-space-y-6"
+            >
+              <li
+                v-for="navbar in subMenus"
+                :key="navbar"
+                class="tw-relative tw-group"
+              >
+                <a
```

Comment:
When `open source` is not present, hovering shows like below

![Screenshot from 2023-06-20 11-47-03](https://github.com/canopas/canopas-website/assets/69897605/3ecae9df-2125-4024-b273-7398abe4aceb)